docs/secrets.md

```diff
@@ -5,33 +5,60 @@ passwords, OAuth tokens, and ssh keys.  Putting this information in a `secret`
 is safer and more flexible than putting it verbatim in a `pod` definition or in
 a docker image.
 
-### Creating and Using Secrets
-To make use of secrets requires at least two steps:
-  1. create a `secret` resource with secret data
-  1. create a pod that has a volume of type `secret` and a container
-     which mounts that volume.
+## Overview of Secrets
+
+
+Creation of secrets can be manual (done by the user) or automatic (done by
+automation built into the cluster).
+
+A secret can be used with a pod in two ways: either attached to one or more of
```

Comment:
'attached to' reads a little weird to me -- what about 'mounted in one or more of its volumes'?